testsuite/forge/src/main.rs

```diff
@@ -184,7 +237,7 @@ impl Test for EmitTransaction {
 impl NetworkTest for EmitTransaction {
     fn run<'t>(&self, ctx: &mut NetworkContext<'t>) -> Result<()> {
         let duration = Duration::from_secs(10);
-        let rng = ::rand::rngs::StdRng::from_seed([0; 32]);
+        let rng = (*ctx.core().rng()).clone();
```

Comment:
Can we use https://docs.rs/rand/0.8.4/rand/trait.SeedableRng.html#method.from_rng instead?